PencilCase/PencilCase/scripts/Models/Product.ts

```diff
@@ -30,6 +32,8 @@ export class Product {
             this.WholesaleUnit = product.WholesaleUnit;
             this.ImportWholesalePrice = product.ImportWholesalePrice;
             this.ImportRetailPrice = product.ImportRetailPrice;
+            this.WholesaleCost = product.WholesaleCost;
```

Comment:
Add 2 new fields for product. The wholesale cost and retail cost. This value will be re-calculated and updated every time when importing, because import prices may change, and then we need to re-calculate the cost with existed inventory.